app/src/main/java/com/sci/recipeandroid/feature/personalize/domain/repository/PersonalizeGoalsFakeRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.sci.recipeandroid.feature.personalize.domain.repository
+
+import com.sci.recipeandroid.feature.personalize.data.mapper.mapIconResNameToId
+import com.sci.recipeandroid.feature.personalize.domain.model.PersonalizeGoalsModel
+import com.sci.recipeandroid.feature.personalize.domain.repository.PersonalizeGoalsFakeRepositoryImpl.RawGoalData
+
+interface PersonalizeGoalsFakeRepository {
```

Comment:
interface name should be PersonalizeGoalsRepository

Reply:
Updated in 4b9217e98b91d1f461395d2ec7cf4637c9e2c293